### ISSUE TITLE
Option -v not recognised by run

### ISSUE THREAD

**Issue by viraptor** [python-poetry/poetry#2266]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [ ] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Fedora 31
- **Poetry version**: 1.0.5
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: 

## Issue

`poetry run` does not recognise the global option `-v` even though it's listed in the documentation:

```
$ poetry help run
USAGE
  poetry run <args1> ... [<argsN>]
...
GLOBAL OPTIONS
  -v (--verbose)         Increase the verbosity of messages: "-v" for normal output, "-vv" for more verbose output and "-vvv" for debug
```

If I try to use it, I just get the help message again for the help command itself:

```
$ poetry -v run pserve development.ini
USAGE
  poetry help [<command1>] ... [<commandN>]
...
```

**Comment by pawamoy** [comment]
Additionally, when using `poetry run -v PROG ARGS`, Poetry will fail with `FileNotFoundError`, since it's trying to execute `-v`.

**Comment by n-batalha** [comment]
The same applies to `install`:

```
$ poetry -vv install 
USAGE
  poetry help [<command1>] ... [<commandN>]
```

Using poetry version 1.0.7 (tried 1.0.5 too). The same applies to any verbose level.

This is an issue as I'd like to understand why an install seems never ending.

**Comment by finswimmer** [comment]
@n-batalha in your case you can use `poetry install -vv`.

**Comment by n-batalha** [comment]
@finswimmer I get a Python exception which I'll perhaps log separately, but thanks! In any case you may want to look into it as `poetry -vv install` is a valid invokation according to docs.

**Comment by sinoroc** [comment]
I somewhat understand why supporting `poetry run -v something` is complicated and/or not welcome. But I must say, I never understood why the global options can not be placed before the command, so that `poetry -v run something` should be possible. It would make sense to place global flags before any specific command. I do not even know if that is true or not, I kind of gave up on trying to understand the CLI logic, it is frustratingly inconsistent to me (I know these things are deceptively hard to handle correctly):

```
$ poetry -v cache list
USAGE
[...]
$ poetry cache -v list
USAGE
[...]
$ poetry cache -vvv list
Too many arguments.
$ poetry cache list -v
pypi
```

**Comment by finswimmer** [comment]
Hello,

I believe this is something that has to be changed in [cleo](https://github.com/sdispater/cleo) and not in poetry itself.

fin swimmer

**Comment by IceTDrinker** [comment]
Any pointers as to where to look to fix this ?

**Comment by branchvincent** [comment]
Thanks for the issue. This has been addressed on `master` and will be released with the upcoming `1.2`:

```console
$ poetry --version
Poetry (version 1.2.0a2)
$ poetry -vvv run echo test
Loading configuration file /Users/Branch/Library/Application Support/pypoetry/config.toml
Using virtualenv: /Users/Branch/Code/python-poetry/poetry/.venv
test
$ poetry run -vvv echo test
Loading configuration file /Users/Branch/Library/Application Support/pypoetry/config.toml
Using virtualenv: /Users/Branch/Code/python-poetry/poetry/.venv
test
```
